### PR TITLE
feat(pack): add mention of the debit on migration pack and move offer

### PR DESCRIPTION
**Pull request by Steffy29**
| Question         | Answer
| ---------------- | ---
| Branch?          | `release/components-w8`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #UXCT-463
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~Breaking change is mentioned in relevant commits~

## Description

Add mention of the debit for migration / move offers on Internet Access

## Related

#UXCT-463

## i18n

- [x] TRANS-49737

### CODE REVIEW

File: packages/manager/apps/telecom/src/app/telecom/pack/move/offers/move-offers.html [hunk 1]
```diff
@@ -898,6 +898,14 @@
                         </tr>
                     </tfoot>
                 </table>
+                <div>
+                    <div
+                        data-translate="pack_move_offers_debit_mention_1"
+                    ></div>
+                    <div
+                        data-translate="pack_move_offers_debit_mention_2"
+                    ></div>
```
Comment (anooparveti) [inline packages/manager/apps/telecom/src/app/telecom/pack/move/offers/move-offers.html:907]:
```suggestion
                    <p
                        data-translate="pack_move_offers_debit_mention_1"
                    ></p>
                    <p
                        data-translate="pack_move_offers_debit_mention_2"
                    ></p>
```

File: packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.html [hunk 2]
```diff
@@ -1005,6 +1005,14 @@
                             </tr>
                         </tfoot>
                     </table>
+                    <div>
+                        <div
+                            data-translate="telecom_pack_migration_offers_debit_mention_1"
+                        ></div>
+                        <div
+                            data-translate="telecom_pack_migration_offers_debit_mention_2"
+                        ></div>
```
[the remark this answers is not in the document]
Comment (Steffy29) [reply packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.html:1014]: Sorry but if I use `<p>` tag the displayed result it's not the result I want to display.

File: packages/manager/apps/telecom/src/app/telecom/pack/move/offers/move-offers.html (hunk 1 above)
Comment (Steffy29) [reply packages/manager/apps/telecom/src/app/telecom/pack/move/offers/move-offers.html:907]: Sorry but if I use `<p>` tag the displayed result it's not the result I want to display.

Comment (anooparveti) [reply packages/manager/apps/telecom/src/app/telecom/pack/move/offers/move-offers.html:907]: @Steffy29 what is the issue?

Comment (Steffy29) [reply packages/manager/apps/telecom/src/app/telecom/pack/move/offers/move-offers.html:907]: the space between the 2 lines has a different size between `<p>` tag and `<div>` tag. I can try another solution if you want.

Comment (anooparveti) [reply packages/manager/apps/telecom/src/app/telecom/pack/move/offers/move-offers.html]: Hello @Steffy29, can you try `<p class="mb-0"></p>`?

File: packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.html [hunk 3]
```diff
@@ -1005,6 +1005,15 @@
...
                     </table>
+                    <div>
+                        <p>
+                            {{'telecom_pack_migration_offers_debit_mention_1' |
+                            translate}}
+                            <br />
+                            {{'telecom_pack_migration_offers_debit_mention_2' |
+                            translate}}
+                        </p>
+                    </div>
```
Comment (anooparveti) [inline packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.html:1016]:
```suggestion
                        <p class="mb-0" data-translate="telecom_pack_migration_offers_debit_mention_1"></p>
                        <p class="mb=0" data-translate="telecom_pack_migration_offers_debit_mention_2"></p>
```

File: packages/manager/apps/telecom/src/app/telecom/pack/move/offers/move-offers.html [hunk 4]
```diff
@@ -898,6 +898,13 @@
                         </tr>
                     </tfoot>
                 </table>
+                <div>
+                    <p>
+                        {{'pack_move_offers_debit_mention_1' | translate}}
+                        <br />
+                        {{'pack_move_offers_debit_mention_2' | translate}}
+                    </p>
+                </div>
```
Comment (anooparveti) [inline packages/manager/apps/telecom/src/app/telecom/pack/move/offers/move-offers.html:907]: Same as https://github.com/ovh/manager/pull/8764/files#r1085391390

File: packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.html (hunk 3 above)
Comment (Steffy29) [reply packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.html:1016]: Sorry but I want to understand what does your method bring more than mine? Better performance, better accessibility,..? I don't understand why you insist on this point.

Comment (anooparveti) [reply packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.html:1016]: Hello @Steffy29, `div` element is a container where `p` is for displaying text.

**Review by anooparveti — DISMISSED** [review]

**Review by ghyenne — DISMISSED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Pull request merged**